cypress/integration/FileInputFieldWithList/Files_can_be_removed/index.js

```diff
@@ -0,0 +1,23 @@
+import '../common'
+import { When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+When('the remove handle behind a file is clicked', () => {
+    cy.contains('test1.md').then($label => {
+        $label
+            .siblings('[data-test="dhis2-uicore-filelistitem-remove"]')
+            .click()
+    })
```

Comment:
Same as above.
If cypress doesn't offer `cy.get(...).siblings(...)`, you can always use [`cy.get(...).invoke(...)`](https://docs.cypress.io/api/commands/invoke.html#Syntax)